db/migrate/20200703001806_create_join_table_plans_diseases.rb

```diff
@@ -0,0 +1,7 @@
+class CreateJoinTablePlansDiseases < ActiveRecord::Migration[5.2]
+  def change
+    create_join_table :plans, :diseases, table_name: :plan_diseases do |t|
+      t.index [:plan_id, :disease_id], unique:true
+    end
```

Comment:
The unique index is good, but it doesn't enforce existence does it? I think it doesn't, could we add foreign key constraints here? See other migrations for examples.